src/app/api/getDelegations.ts

```diff
@@ -57,12 +58,200 @@ export const getDelegations = async (
     staker_btc_pk: encode(publicKeyNoCoord),
   };
 
-  const response = await apiWrapper(
-    "GET",
-    "/v1/staker/delegations",
-    "Error getting delegations",
-    params,
-  );
+  let response;
+  try {
+    response = await apiWrapper(
+      "GET",
+      "/v1/staker/delegations",
+      "Error getting delegations",
+      params,
+    );
+  } catch (error) {
+    console.error(error);
+    response = {
+      data: {
+        delegations: [
```

Comment:
When I suggested adding tests, I meant unit and/or integration tests that test component behavior. This could be as simple as testing that the data is rendered in the table with the appropriate fields. This would be really impactful once you get to unbonding so we can mock the contract calls and ensure that all invariants are handled. 

This is fine for now, but I think this should be put into a separate test data directory so that we don't pollute this file with extraneous stuff.